score/etl/sources/census/etl.py

```diff
@@ -0,0 +1,115 @@
+import csv
+import os
+import json
+from pathlib import Path
+import logging
+
+from .etl_utils import get_state_fips_codes
+from utils import unzip_file_from_url
+
+
+def download_census_csvs(data_path: Path) -> None:
+    # the fips_states_2010.csv is generated from data here
+    # https://www.census.gov/geographies/reference-files/time-series/geo/tallies.html
+    state_fips_codes = get_state_fips_codes(data_path)
+    for fips in state_fips_codes:
+        # check if file exists
+        shp_file_path = data_path.joinpath(
```

Comment:
Any reason we've switched to joinpath over just using slashes (e.g., `data_path / "census" / "shp" / ...`)? 

I don't have a strong preference, in all things I just prioritize consistency.

Reply:
Fixed.